base/abstractarray.jl

```diff
@@ -5,8 +5,9 @@
 """
     AbstractArray{T,N}
 
-Abstract `N`-dimensional array supertype containing elements of type `T`.
-[`Array`](@ref)s and some array-like objects inherit from this type.
+Supertype for `N`-dimensional arrays (or array-like types) with elements of type `T`.
+[`Array`](@ref) and other types inherit from this. See the manual section on the
```

Comment:
maybe "are subtypes of this"? since the meaning of inherit isn't necessarily clear